### DISCUSSION TITLE
Is there a limit to how many Input Field Templates that can be added via the plugin settings?

### DISCUSSION THREAD

**Discussion by Insight-Rich** [mProjectsCode/obsidian-meta-bind-plugin#427] [Q&A] [answered]
I seem to be stuck at defining 3 Input Field Templates in the plugin settings.  Once I have 3 defined, no more will appear when I click the "Add Template" button.  Something is evidently happening, because if I then click "Save", then I get a "Some Templates Filed to Parse" error:

![image](https://github.com/user-attachments/assets/0ee7bad3-d422-401d-b21f-ddecd93e5d4a)

Something very similar also happens with Button Templates.  

I suspect that the problems are to do with the "Edit Templates" dialogs, because if I manually add a 4th template to the data.json file in ".obsidian\plugins\obsidian-meta-bind-plugin" then it works fine, and all 4 will appear in the dialog.

Am I doing something wrong here, or should this be submitted as a bug?

**Comment by mProjectsCode** [comment, marked as answer]
It's a bug. There will be a fix soon.

**Reply by Insight-Rich** [reply]
Awesome - just updated to 1.2.1 and it seems to work perfectly!  Thanks for the plugin, and thanks for the speedy fix! =]
